finetune/target_models/masked_language_model.py

```diff
@@ -96,6 +98,34 @@ def __init__(self, *args, **kwargs):
     def _get_input_pipeline(self):
         return MaskedLanguageModelPipeline(self.config)
 
+    def predict_top_k_report(self, input_text, k=5, context=None, **kwargs):
+        prediction_info = self._inference(
+                [input_text],
+                predict_keys=[
+                    PredictMode.GENERATE_TEXT,
+                    PredictMode.MLM_IDS,
+                    PredictMode.MLM_POSITIONS],
+                context=context,
+                force_build_lm=True,
+                **kwargs)
+
+        predicted_tokens = [{'prediction_ids': [self.input_pipeline.text_encoder.decode([i]) for i in pred['GEN_TEXT'][-k:][::-1]],
```

Comment:
These keys should probably use the enum and not the string values.

Reply:
Alright, updated.